Fix Windows CI Builds

Windows CI builds are currently failing on AppVeyor with:
```
"C:\projects\nas2d-core\proj\vc14\NAS2D.sln" (default target) (1) ->
"C:\projects\nas2d-core\proj\vc14\NAS2D.vcxproj" (default target) (2) ->
(ClCompile target) -> 
  ..\..\src\Filesystem.cpp(226): error C3861: 'PHYSFS_readBytes': identifier not found [C:\projects\nas2d-core\proj\vc14\NAS2D.vcxproj]
  ..\..\src\Filesystem.cpp(267): error C2065: 'PHYSFS_Stat': undeclared identifier [C:\projects\nas2d-core\proj\vc14\NAS2D.vcxproj]
  ..\..\src\Filesystem.cpp(267): error C2146: syntax error: missing ';' before identifier 'stat' [C:\projects\nas2d-core\proj\vc14\NAS2D.vcxproj]
  ..\..\src\Filesystem.cpp(268): error C3861: 'PHYSFS_stat': identifier not found [C:\projects\nas2d-core\proj\vc14\NAS2D.vcxproj]
  ..\..\src\Filesystem.cpp(268): error C2228: left of '.filetype' must have class/struct/union [C:\projects\nas2d-core\proj\vc14\NAS2D.vcxproj]
  ..\..\src\Filesystem.cpp(268): error C2065: 'PHYSFS_FILETYPE_DIRECTORY': undeclared identifier [C:\projects\nas2d-core\proj\vc14\NAS2D.vcxproj]
  ..\..\src\Filesystem.cpp(349): error C3861: 'PHYSFS_writeBytes': identifier not found [C:\projects\nas2d-core\proj\vc14\NAS2D.vcxproj]
  ..\..\src\Filesystem.cpp(434): error C3861: 'PHYSFS_getLastErrorCode': identifier not found [C:\projects\nas2d-core\proj\vc14\NAS2D.vcxproj]
  ..\..\src\Filesystem.cpp(434): error C3861: 'PHYSFS_getErrorByCode': identifier not found [C:\projects\nas2d-core\proj\vc14\NAS2D.vcxproj]
    4 Warning(s)
    9 Error(s)
```

This is caused by the recent PhysFS upgrade. The methods are available in the PhysFS 3.0.1 release which the code is now targetting. These methods are not available in the old PhysFS 2.0.3 package, which the CI build is still installing.

The changes were introduced during an AppVeyor service interruption due to a configuration problem on our end.

----

Long term, there seems to be some desire to move away from PhysFS, and towards the new native C++17 `<filesystem>` library. Due to slow adoption and old tools on some CI providers, such an upgrade is currently blocked.

For now, I think we should try to update the build dependencies on the AppVeyor build to use PhysFS 3.0.1.

The current config uses NuGet packages to install the dependencies. Currently NuGet repositories offer a PhysFS 2.0.3 package, but have no available PhysFS 3.0.1 package.

We could potentially try to put together a NuGet package for PhysFS 3.0.1. I believe the source code licensing should allow use to create such a package ourselves. It might also be of interest to the PhysFS author. Microsoft has documentation on [Creating NuGet packages](https://docs.microsoft.com/en-us/nuget/create-packages/creating-a-package).

Failing that, we could create a script to download the package on the AppVeyor build.

Alternatively, we could potentially rollback the changes for the PhysFS 3.0.1 upgrade. That may be difficult to undo the piecemeal updates though. There are likely quite a few dependent changes already merged on top of them by now.

--- a/.appveyor.yml
+++ b/.appveyor.yml
@@ -5,11 +5,11 @@ platform:
   - x64
 cache:
   - C:\tools\vcpkg\installed\
-  - C:\projects\nas2d-core\proj\vc14\packages -> proj\vc14\packages.config
+  - C:\projects\nas2d-core\proj\vs2017\packages -> proj\vs2017\packages.config
 install:
   - vcpkg integrate install
   - vcpkg install physfs:x86-windows
   - vcpkg install physfs:x64-windows
-  - nuget restore proj/vc14
+  - nuget restore proj/vs2017
 build:
-  project: proj/vc14/NAS2D.sln
+  project: proj/vs2017/NAS2D.sln

--- a/proj/vs2017/NAS2D.vcxproj
+++ b/proj/vs2017/NAS2D.vcxproj
@@ -305,7 +305,6 @@
   </ItemGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
-    <Import Project="packages\physfs.v140.2.0.3.1\build\native\physfs.v140.targets" Condition="Exists('packages\physfs.v140.2.0.3.1\build\native\physfs.v140.targets')" />
     <Import Project="packages\glew.redist.1.9.0.1\build\native\glew.redist.targets" Condition="Exists('packages\glew.redist.1.9.0.1\build\native\glew.redist.targets')" />
     <Import Project="packages\glew.1.9.0.1\build\native\glew.targets" Condition="Exists('packages\glew.1.9.0.1\build\native\glew.targets')" />
     <Import Project="packages\sdl2.redist.2.0.5\build\native\sdl2.redist.targets" Condition="Exists('packages\sdl2.redist.2.0.5\build\native\sdl2.redist.targets')" />
@@ -324,7 +323,6 @@
     <PropertyGroup>
       <ErrorText>This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
     </PropertyGroup>
-    <Error Condition="!Exists('packages\physfs.v140.2.0.3.1\build\native\physfs.v140.targets')" Text="$([System.String]::Format('$(ErrorText)', 'packages\physfs.v140.2.0.3.1\build\native\physfs.v140.targets'))" />
     <Error Condition="!Exists('packages\glew.redist.1.9.0.1\build\native\glew.redist.targets')" Text="$([System.String]::Format('$(ErrorText)', 'packages\glew.redist.1.9.0.1\build\native\glew.redist.targets'))" />
     <Error Condition="!Exists('packages\glew.1.9.0.1\build\native\glew.targets')" Text="$([System.String]::Format('$(ErrorText)', 'packages\glew.1.9.0.1\build\native\glew.targets'))" />
     <Error Condition="!Exists('packages\sdl2.redist.2.0.5\build\native\sdl2.redist.targets')" Text="$([System.String]::Format('$(ErrorText)', 'packages\sdl2.redist.2.0.5\build\native\sdl2.redist.targets'))" />

--- a/proj/vs2017/packages.config
+++ b/proj/vs2017/packages.config
@@ -4,7 +4,6 @@
   <package id="glew.redist" version="1.9.0.1" targetFramework="native" />
   <package id="ikkepop.sdl2_ttf" version="2.0.14" targetFramework="native" />
   <package id="ikkepop.sdl2_ttf.redist" version="2.0.14" targetFramework="native" />
-  <package id="physfs.v140" version="2.0.3.1" targetFramework="native" />
   <package id="sdl2" version="2.0.5" targetFramework="native" />
   <package id="sdl2.new" version="2.0.8" targetFramework="native" />
   <package id="sdl2.new.redist" version="2.0.8" targetFramework="native" />

